packages/core/react/src/WalletProvider.tsx

```diff
@@ -76,10 +76,7 @@ export function WalletProvider({
         }
         return [mobileWalletAdapter, ...adaptersWithStandardAdapters];
     }, [adaptersWithStandardAdapters, mobileWalletAdapter]);
-    const [walletName, setWalletName] = useLocalStorage<WalletName | null>(
-        localStorageKey,
-        getIsMobile(adaptersWithStandardAdapters) ? SolanaMobileWalletAdapterWalletName : null
-    );
+    const [walletName, setWalletName] = useLocalStorage<WalletName | null>(localStorageKey, null);
     const adapter = useMemo(
         () => adaptersWithMobileWalletAdapter.find((a) => a.name === walletName) ?? null,
```

Comment:
One possibility for this: if the _only_ item in `adaptersWithMobileWalletAdapter` is MWA, could we select it by default?

This preserves the existing behavior if the user is on Android mobile (so no other wallets are being injected), the app is passing `wallets={[]}`, and no other Standard Wallets are registered. I don't know how common this is, but it might not hurt?